src/store/ViewStore.js

```diff
@@ -78,6 +101,9 @@ export default class ViewStore {
     }
 
     inputRef = (ref) => {
-        if (ref) { ref.focus(); }
+        if (ref) {
+            ref.value = this.templateName;
```

Comment:
you should not need to do this. Consider using a controlled component https://reactjs.org/docs/forms.html#controlled-components